HW2

tzatushevskaya opened the pull request:
- Added image
 - Added homework1.html file to new directory

Commit:
- Added homework1.html file to new directory
 - Added image

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+                    <li>- A page contains different ‘div’ sections. Each one starts with corresponding heading and ends with ‘hr’</li>
+                    <li>- Section 1. Copy this homework’s task and represent it as italic text. Preserve structure (lists) and line breaks</li>
+                    <li>- Section 2. Enumerate your items in a list. Name of each item should be a link to an article in the internet. A link should open in a new tab</li>
+                    <li>- Section 3. Pick one of items and provide some information for it, in 2-3 sentences in a paragraph. Highlight 3 important words/phrases with special color, ‘underline’ and ‘bold’ styles, accordingly. Put the paragraph in a new ‘div’ with maximum width 300px (set this style via CSS)</li>
```
![pic1](https://user-images.githubusercontent.com/16742190/73011755-8e43c080-3e1d-11ea-9cd9-7aa6b9adcdb6.JPG)
Please, check the screenshot. Many symbols are displayed incorrectly (Chrome browser). 
Looks like a good opportunity to use `<meta>` tag with encoding provided:
https://www.w3schools.com/tags/att_meta_charset.asp

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+<hr/>
+<div>
+    <ol>
+        <li><a href="https://en.wikipedia.org/wiki/Scala_(programming_language)">Scala</a></li>
```
According to the requirements, a link should open in a new tab. Please, check '_target_' attribute:
https://www.w3schools.com/tags/att_a_target.asp

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+<button id="myId" onclick="alllert()">Check if you are a bad tester!</button>
+</body>
+<script>
+    function alllert(){
```
Please, name the function with a more accurate name and follow the task

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+        alert ('You are!')
+    }
+</script>
+</html>
```
As you see, 'Github' has marked this line with a 'warning' icon. It is suggested to always have an empty line at the end of a file.
You may automate this, - please, enable this configuration in your IDE:
https://stackoverflow.com/questions/36019534/add-an-empty-line-at-end-of-file-according-to-psr-2-on-phpstorm

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+        }
+        #myId{
+            color: tomato;
+            background-color: cadetblue;
```
I suggest you to not change background color of a simple button, - a default one looks more natural.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+        div {
+            background-color: dodgerblue;
+        }
+        #myId{
```
Please, provide accurate and clear names to IDs, classes, etc.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+        #myId{
+            color: tomato;
+            background-color: cadetblue;
+            size: A3;
```
Doesn't look like this property really affects anything on the page. 
If you want to change size of text, better use `font-size` property.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+                        Add 2 input fields with placeholders ‘a’ and ‘b’ accordingly.<br>
+                        Add button [Hypotenuse](should calculate hypotenuse from values, provided in the inputs and output result in an  ‘alert’ window).<br>
+                    </li>
+                    <li>- Section 5. Pick a palette, make picture of it, put it in the same directory as the HTML file and display it in this section of the page. Add ‘Palette’ title to the image</li>
```
Please, implement this point as well.
Don't forget about `title` attribute.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+                    <li>- Section 1. Copy this homework’s task and represent it as italic text. Preserve structure (lists) and line breaks</li>
+                    <li>- Section 2. Enumerate your items in a list. Name of each item should be a link to an article in the internet. A link should open in a new tab</li>
+                    <li>- Section 3. Pick one of items and provide some information for it, in 2-3 sentences in a paragraph. Highlight 3 important words/phrases with special color, ‘underline’ and ‘bold’ styles, accordingly. Put the paragraph in a new ‘div’ with maximum width 300px (set this style via CSS)</li>
+                    <li>- Section 4. Add button [Name](should output your first name in console).<br>
```
Please, implement this point as well.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+            <li>Requirements:
+                <ul>
+                    <li>- ‘Homework 1’ is title of the page</li>
+                    <li>- A page contains different ‘div’ sections. Each one starts with corresponding heading and ends with ‘hr’</li>
```
Please, implement this point as well.

eclimov replied:
Don't forget about heading in each section

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+            background-color: cadetblue;
+            size: A3;
+        }
+        .myClass{
```
Please, provide accurate and clear names to IDs, classes, etc.
'_section1_' class name would be better in this case.

eclimov replied:
You may also provide a class name or an ID to each section, - it would make the HTML more clear to a reader.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+    <br/>The technology involves a holistic programming language, structured editor and infrastructure for building backend web services.
+    <br/>“We keep adding things, but we never take things away. AWS has 200 services and never takes any away,” he said, calling garbage collection the last really good innovation in the industry.
+</div>
+<button id="myId" onclick="alllert()">Check if you are a bad tester!</button>
```
Don't forget to put your buttons in a separate `div`.

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+            size: A3;
+        }
+        .myClass{
+            width: 300px;
```
According to the task, you should set **max width** of the div

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+<hr/>
+<div class="myClass">
+    <br/>There are too many incrementalists in tech these days, according to Paul Biggar, who along with Ellen Chisa, is unveiling the Dark, a programming language aimed at making deploying software simpler.
+    <br/>“The <b>complexity</b> of building backends today is just out of control. It’s infrastructure, it’s deployment, it’s APIs… Dark is about removing all this complexity,” Biggar said.
```
According to the task, it is required to 

> Highlight 3 important words/phrases with special color, ‘underline’ and ‘bold’ styles, accordingly

You have used only bold style at the moment. Please, apply the remaining styles somewhere in the text as well.

eclimov reviewed (changes requested):
Also, please, remove/replace that 'image.jpg', as it is not part of the task.

eclimov commented:
Actually, it is **homework 2**, - please also do the following, according to the requirements:

> Update 'README.md' with task of previous homework

Commit:
- Added target tag to links
 - Set up empty line automated adding to the end of file
 - Completed homework1
 - Removed image
 - Renamed id-s
 - Added div-s

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,72 @@
+            background-color: cadetblue;
+            size: A3;
+        }
+        .myClass{
```
Please, remove unused class

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,112 @@
+    </ol>
+</div>
+<hr/>
+<div  id="section_3">
```
This section should have maximum width 300px

eclimov commented on public/Sjofn/homework1.html:
```diff
@@ -0,0 +1,112 @@
+    <button id="myId" onclick="console.log('Tanya')">Name</button>
+</div>
+<hr/>
+<div  id="section_5">
```
This section should contain only a picture of palette

eclimov replied:
Looks like you forgot to add and commit the picture. Please, check.

Commit:
- Added titles
 - Updated colours
 - Added picture of the palette
 - Removed myClass

Commit:
Merge remote-tracking branch 'origin/master' into sjofn-hw2

Commit:
- Updated README.md: added Homework1

Commit:
- Changed text color on Section5

Commit:
- Added image to github
 - Added HTML formatting

Commit:
- Made everything perfect
 - Added HTML formatting AGAIN

Commit:
Merge remote-tracking branch 'origin/master' into sjofn-hw2

# Conflicts:
#	README.md

eclimov reviewed (approved):
Well done!

SandaPanteleimonov reviewed (commented):
This is brilliant! Approved 100!
